### DISCUSSION TITLE
Flash of ESPcam - Reflection

### DISCUSSION THREAD

**Discussion by svenschedlbauer** [jomjol/AI-on-the-edge-device#2588] [Q&A]
Hey all,
I have everytime problem with the reflection of the flash from the ESPcam. What solutions do you have? I have already tried to use different tapes, but it help not enough. Sometime I see pictures online from watermeter without any reflections. How does it work. Hope anybody can help me.

**Comment by caco3** [comment]
search the discussions for Diffusor: https://github.com/jomjol/AI-on-the-edge-device/discussions?discussions_q=diffusor+
